### DISCUSSION TITLE
How to reset password without 2FA

### DISCUSSION THREAD

**Discussion by devagekarswati** [community/community#116656] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

HOW TO RESET PASSWORD WITHOUT 2FA

I am unable to reset my password without 2FA. I do have authentiction code or recovery codes. How do i reset my password without 2FA

**Comment by cfprocha** [comment]
Without 2FA I know of two methods you can use. None of them are fast:

1. Using a verified device, SSH token, or personal access token: you can apply for account recovery by proving that you are logging in from a device you have used before, or using a SSH or personal access token that has been used before. The request will be then manually considered by GitHub support;
2. Using an account recovery token on Facebook (if you have set it previously): This also requires manual verification.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @devagekarswati ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by devagekarswati** [reply]
I want GitHub support team to delete repository from my account "swatidevagekar" as I am unable to login. Could you please help me to raise support request for deletion of repository from account .

**Comment by devagekarswati** [comment]
Hi Carlos,

1. I have accidentally deleted the authenticator code from authentication
app and do not have any SSH token or personal token so I cannot try the
first option.

2. I have not used Facebook previously for GitHub.

Could you guide me any other way?

Can u help me to delete the repository I want to?

Thanks,
Swati

**Comment by devagekarswati** [comment]
Hi Carlos,

Any update on this? It is on high priority right now. Could you please help
here.

Thanks,
Swati

**Comment by git-sumana** [comment]
https://github.com/password_reset
Reset here by adding a new password after entering the email address

**Reply by devagekarswati** [reply]
I already tried that option. As i have lost 2FA & unable to recover it i cannot reset password. So i need github support team to delete the client repo which is by mistake hosted publicly.

**Reply by git-sumana** [reply]
Once try "Contact Support" or search in gmail regarding the recovery codes, only that may help you..

**Reply by devagekarswati** [reply]
Could you please help me with github support email address?
